utils/score_sclite_wo_dict.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright 2019 Okayama University (Katsuki Inoue)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+[ -f ./path.sh ] && . ./path.sh
+
+wer=false
```

Comment:
How about exposing the option to run.sh? WER-based evaluations would be also useful.

Reply:
I added the option of WER evaluation.